sos/report/plugins/dnf.py

```diff
@@ -120,8 +126,9 @@ def setup(self):
         self.get_modules_info(modules['output'])
 
     def postproc(self):
-        regexp = r"(proxy_password(\s)*=(\s)*)(\S+)\n"
+        regexp = r"(password(\s)*=(\s)*)(\S+)\n"
```

Comment:
Does this need a leading `(.*)?` optional match in order to still hit `proxy_password` along with all others?

Reply:
Until we have there `^` or something similar, it should work well - as I checked:

    # cat /etc/dnf/vars/test.conf 
    proxy_password = test
    password = different
    # python3 bin/sos report -o dnf --batch --build
    ..
    # cat /var/tmp/sosreport-pmoravec-rhel8-2022-09-21-vfswezo/etc/dnf/vars/test.conf 
    proxy_password = ********
    password = ********
    #